crates/ruff/resources/test/fixtures/flake8_implicit_str_concat/ISC.py

```diff
@@ -59,3 +59,23 @@
 _ = foo + bar + "abc"
 _ = "abc" + foo + bar
 _ = foo + "abc" + bar
+
+# Multiple strings nested inside a f-string
+_ = f"a {'b' 'c' 'd'} e"
+_ = f"""abc {"def" "ghi"} jkl"""
+_ = f"""abc {
+    "def"
+    "ghi"
+} jkl"""
```

Comment:
We didn't detect this in earlier version but with the granularity of the tokens, we can detect this as well.